hansol/app.js

```diff
@@ -0,0 +1,76 @@
+const http = require('http')
+
+const express = require('express')
+const morgan = require('morgan')
+const cors = require('cors')
+const dotenv = require('dotenv')
+const { DataSource } = require('typeorm')
+dotenv.config()
+
```

Comment:
한솔님 ! module export 시 아래와 같은 순서로 수정해서 작성해 주세요 !

```suggestion
require('dotenv').config()

const http = require('http')
const express = require('express')
const morgan = require('morgan')
const cors = require('cors')
const { DataSource } = require('typeorm')
```